parks.html

```diff
@@ -0,0 +1,9 @@
+
+
+
+
+
+
+
+
+<script src="scripts/parks_page.js" ></script>
```

Comment:
This is really great code, good idea!

Reply:
Why did you choose to name this "parks_page" instead of "parks"?

Reply:
I like parks_page!